holidays/countries/india.py

```
@@ -557,7 +603,20 @@ def _populate_subdiv_kl_public_holidays(self):
 
     # Ladakh.
     def _populate_subdiv_la_public_holidays(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_populate_subdiv_la_public_holidays`

Add return type annotation: `None`

(ANN202)